setup/default.xml

```diff
@@ -76,6 +76,10 @@
         SELECT * FROM tc_statistics WHERE captureTime BETWEEN :from AND :to ORDER BY captureTime
     </entry>
 
+    <entry key='totp.enabled'>false</entry>
```

Comment:
If these are defaults, please remove from this file.

Reply:
removed as false is the default. Documentation needs to be updated for these new settings